File: python/quick-start/outbound_call.py
import os

from dotenv import load_dotenv
from phonic import Phonic

load_dotenv(".env.local")

client = Phonic(
    api_key=os.getenv("PHONIC_API_KEY"),
)

client.conversations.outbound_call(
    to_phone_number=os.getenv("CUSTOMER_PHONE_NUMBER", "") # e.g. +15551234567
    config={
        "agent": "my-first-agent",
        "welcome_message": "Hello, how can I help you?",
    },
)

File: node/quick-start/outbound-call.ts
import { config } from "dotenv";
import { PhonicClient } from "phonic";

config({ path: ".env.local" });

const client = new PhonicClient({
  apiKey: process.env.PHONIC_API_KEY,
});

async function outboundCall() {
  await client.conversations.outboundCall({
    to_phone_number: process.env.CUSTOMER_PHONE_NUMBER as string,
    config: {
      agent: "my-first-agent",
    },
  });
}

outboundCall();
